db/migrate/20220126052157_change_segment_cjk_site_setting.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class ChangeSegmentCjkSiteSetting < ActiveRecord::Migration[6.1]
+  def up
+    execute <<~SQL
+    UPDATE site_settings
+    SET name = 'search_tokenize_chinese'
```

Comment:
Note that we're defaulting `search_tokenize_japanese` to false after this migration because the previous site setting has not worked for japanese before. It is safe to assume that people who turned on the site setting previously did it for the chinese language.